front-end/src/components/cards/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import {Container} from './styled'
+
+const Card = ({numberCard, titleCard, gameVariable, imgNaipeCard}) => {
+    return (
+        <Container> 
+            <div className="wrapper-content">
+                <div className="top-card">
+                    <span>{numberCard}</span>
+                    <img src={imgNaipeCard} />
+                </div>
+                <div className="content-card">
+                    <span className="span-text">{titleCard}</span>
+                    <span className="span-number">{gameVariable}</span>
+                </div>
+                <div className="bottom-card">
+                    <div className="div-naipe">
+                        <img src={imgNaipeCard} />
```

Comment:
Ow, é chatão ficar cheio de erro no terminal kk, só coloca um "alt" aí na tag img, é realmente uma boa prática, de vdd!

```suggestion
                        <img src={imgNaipeCard} alt="Naipe da carta" />
```